### PR TITLE
Update page in Connect AWS Gateway “prepare-aws-api-gateway”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
Jira: APIGOV-16135

**Comment by jcollins-axway** [comment]
@kflan-io @sbolosan @dfeldick @vivekschauhan - please review content

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md [hunk 1]
```diff
@@ -29,52 +29,147 @@ This zip contains three files:
...
 
-Upload the yaml files from the `cloudformation-continuous-discovery.zip` archive and the `traceability_lambda.zip` archive to an AWS S3 bucket located on the same region the Cloud Formation template will be deployed.
+Create a new bucket on S3 and upload the following (note the bucket name and URL to the `amplify-agents-deploy-all.yaml` file).
+
+* Upload the yaml files from the `cloudformation-continuous-discovery.zip` archive and the `traceability_lambda.zip` archive to an AWS S3 bucket located on the same region the Cloud Formation template will be deployed.
+
+* For deployment within an EC2 instance setup the following
 
-Note the bucket name where these are uploaded to and save the URL to the `amplify-agents-deploy-all.yaml` file.
+    * Within this bucket create a folder named **resources** add the following:
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:40]: "," (comma) between bucket and create

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md [hunk 2]
```diff
@@ -29,52 +29,147 @@ This zip contains three files:
...
-
-Now you are ready to [deploy agents](/docs/central/connect-aws-gateway/deploy-your-agents-1).
+* Amplify Agents Stack Options
+
+    * Set the **AgentResourcesBucket** value to the S3 Bucket where you saved all of the files.
+    * Enable **APIGWCWRoleSetup** to create an IAM role for writing API Gateway logs in CloudWatch.
+    * Accept the **APIGWTrafficLogGroupName** default value of `aws-apigw-traffic-logs` to store AWS API Gateway call logs in CloudWatch.
+    * Enable **ConfigServiceSetup** to create an IAM role for using Config Service.
+    * Set the **ConfigBucketExists** value to `true` to use an existing S3 bucket to store the AWS logs. Be sure the S3 bucket is accessible and is located in the same region where you run the Cloud Formation template.
+    * Set the **ConfigBucketName** value to the S3 Bucket where you saved all of the files, to store AWS Config logs. When **ConfigBucketExists** is `false` the accountID and AWS region are appended to the value and the CloudFormation will attempt to create a new bucket.
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:103]: ...the accountID and AWS region are appended to the value. The CloudFormation will attempt to create a new bucket using the newly created name.

Just a suggestion, if I'm thinking that's what you were trying to say.

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md [hunk 3]
```diff
@@ -29,52 +29,147 @@ This zip contains three files:
...
+    * Accept the **ECSCentralOrganizationID** default value of blank, for quickstart.
+    * Accept the **ECSCentralEnvironmentName** default value of blank, for quickstart.
+    * Accept the **ECSCentralClientID** default value of blank, for quickstart.
+
+* EC2/ECS Common Deployment Settings (ignore if **DeploymentType** is `Other`). Suggested values for quickstart.
+
+    * Accept the **DiscoveryAgentLogGroupName** default value of `amplify-discovery-agent-logs` to create a log group for the Discovery agent logs.
+    * Accept the **TraceabilityAgentLogGroupName** default value of `amplify-traceability-agent-logs` to create a log group for the Traceability agent logs.
+    * Set the **SSMPrivateKeyParameter** to the name of the Parameter holding the private_key.pem contents. `AmplifyPrivateKey` by default.
+    * Set the **SSMPublicKeyParameter** to the name of the Parameter holding the private_key.pem contents. `AmplifyPrivateKey` by default.
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:132]: not sure if this is correct, you're the expertise on this... just making sure you really want to set the public key parameter with private key pem and amplify private key...  if this is correct, ignore =)

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 2 above)
Comment (jcollins-axway) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md:103]: Set the **ConfigBucketName** value to the S3 Bucket where you saved all of the files, to store AWS Config logs. When **ConfigBucketExists** is `false` the CloudFormation will attempt to create a new bucket using the name and appended with the account id and region.

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 3 above)
Comment (jcollins-axway) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md]: my mishap, fixed

**Review by sbolosan — APPROVED** [review]

File: content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md (hunk 2 above)
Comment (sbolosan) [reply content/en/docs/central/connect-aws-gateway/prepare-aws-api-gateway.md]: good to me!

**Review by sbolosan — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes,
@lbadenhop please proceed.

**Review by vivekschauhan — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

**Review by kflan-io — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**